salt/utils/cloud.py

```diff
@@ -2780,6 +2797,10 @@ def cache_nodes_ip(opts, base=None):
     Retrieve a list of all nodes from Salt Cloud cache, and any associated IP
     addresses. Returns a dict.
     '''
+    salt.utils.warn_until(
+        'Flourine',
+        'This function is incomplete and non-functional.'
```

Comment:
Can we flesh out this message a little bit more? Maybe just add something like `This function will be removed in Salt Fluorine.` to the end of what you already have. :)